source/jormungandr/jormungandr/street_network/geovelo.py

```diff
@@ -105,11 +105,33 @@ def _make_request_arguments_bike_details(cls, bike_speed_mps):
             'averageSpeed': bike_speed,  # in km/h, BEGINNER sets it to 13
         }
 
+    @classmethod
+    def sort_by_mode(cls, points):
```

Comment:
It seems that this method is critical, a unit test may be useful?

Reply:
:+1: